notebooks/model_explorer_streaming.ipynb

```diff
@@ -0,0 +1,1140 @@
+{
```

Comment:
<p>How are these userdate.get lines supposed to work? Trying this with no prior step in the colab fails, maybe we should have a cell where we ask for user input? Or instructions on how to set the userdata? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Oh actually there's documentation about that in the first text block. I can move that down here to make it more visible.